### DISCUSSION TITLE
Option to have different "Profiles"

### DISCUSSION THREAD

**Discussion by robinradx** [rxhanson/Charmstone-Community#46] [Ideas]
Hey there,

it's a great tool deeply embedded in my workflow (just like Rectangle Pro, thanks for both tools - they are great) there is only one thing that would made it even more awesome to me. I have different "use cases" with a very different need in apps (e.g. studying / private / business / dev). It would be great if there would be an option to choose different "profiles" to assign apps to.

Best,
Rob

**Comment by rxhanson** [comment]
Thanks for the feedback! This is one of the most commonly requested features for Charmstone. I do have plans to implement this, but I don't have a timeframe lined up for it.
